docker/Makefile

```diff
@@ -13,26 +13,30 @@ HEALTHBOT_OVERRIDE=$(shell if [ ${HEALTHBOT_ENABLED} = "true" ]; then echo "-f d
 
 staging-up:
 	PLATFORM=$(shell uname -m) \
-	ENVIROMENT=deployed/staging \
+	USER=$(shell id -u) \
```

Comment:
Note that we set `USER` to the current user. 

This is needed since the Grafana docker image sets some user id that usually doesn't exist in the host. This creates problems with permissions in the data folder to save the state. 

Here we follow official advice of running the container with the user id that is currently running. In the case of our infra would be `validator` which has access in the new `grafana/data` folder.